src/distributions/normal.jl

```diff
@@ -15,88 +15,54 @@ struct Normal{T,U,V} <: AbstractNormal{T}
     Σ::V
 end
 
-Normal{T}(μ, Σ) where {T} = Normal{T,typeof(μ),typeof(Σ)}(μ, Σ)
-
-"""
-    Normal(μ::AbstractVector{<:Real}, Σ::Symmetric{<:Real})
-
-Creates a Normal distribution with mean vector μ and covariance matrix Σ.
-"""
-function Normal(μ::AbstractVector{<:Real}, Σ::Symmetric{<:Real})
-    T = promote_type(eltype(μ), eltype(Σ))
-    return Normal{T}(convert(AbstractVector{T}, μ), convert(AbstractMatrix{T}, Σ))
+function Normal{T}(μ::AbstractVector, Σ, ::IsPSDParametrization) where {T}
```

Comment:
Just to bring it up, an alternative would be to simply dispatch on a static boolean here, e.g. using Static.jl. 
The resulting code would look somewhat like this:
```julia
function Normal{T}(μ::AbstractVector, Σ, is_psd::True) where {T}
```
```julia
function Normal{T}(μ::AbstractVector, Σ, is_psd::False) where {T}
```
where `True` and `False` are imported from Static.jl

Reply:
I think claiming ```ispsd``` to be ```True``` or ```False``` would be misleading as that can generally not be inferred from just type information. Instead its more like trying to discern between types that could 'reasonably' be hypothesized to be psd and those that can not. See also response below for further musings.